airflow/decorators/python.py

```diff
@@ -35,7 +35,7 @@ class _PythonDecoratedOperator(DecoratedOperator, PythonOperator):
     """
 
     template_fields: Sequence[str] = ('op_args', 'op_kwargs')
-    template_fields_renderers = {"op_args": "py", "op_kwargs": "py"}
+    template_fields_renderers = {"op_args": "json", "op_kwargs": "json"}
```

Comment:
This seems wrong to me at first glance. Please include a screenshot of how this is shown in the UI.

Reply:
Hi @ashb you're right it's quite counter-intuitive, I think this is done because args/kwargs are essentially a list/dict.
Here's what it looks like in the UI (based on [avkirilishin](https://github.com/avkirilishin) answer)

**Before** ![](https://user-images.githubusercontent.com/54231417/151705503-63edd4cb-9b82-4466-a201-e07d5eb37008.png)

**After** ![](https://user-images.githubusercontent.com/54231417/151705515-696bf584-5baf-4dcb-b714-87cb7c730793.png)